notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<p>Thank you for going into so much detail about this.</p><p> </p><p>nit: "Calculating the Manhattan distance is easy". Stylistically I recommend erring against using words like "simple" or "easy" as it doesn't always end up being the case for all users. I would instead reword as "To calculate the Manhattan distance, assume that you..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>